packages/blockchain/test/index.ts

```diff
@@ -158,8 +162,12 @@ test('blockchain test', t => {
       st.error(err, 'no error')
       blockchain.getBlock(genesisBlock.hash(), (err?: Error, block?: Block) => {
         st.error(err, 'no error')
-        st.equal(block.hash().toString('hex'), genesisBlock.hash().toString('hex'))
-        st.end()
+        if (block) {
```

Comment:
TSLint fix: checking value of `block?` before trying to access it.

Reply:
nice yeah sometimes if i'm in a test suite i'll just use `!` to reduce adding bloat. if it is undefined it should error out. the error message is nice though.